etc/picongpu/taurus-tud/V100_picongpu.profile.example

```diff
@@ -76,7 +76,34 @@ export CXXFLAGS="-Dlinux"
 export TBG_SUBMIT="sbatch"
 export TBG_TPLFILE="etc/picongpu/taurus-tud/V100.tpl"
 
-alias getNode='srun -p ml --gres=gpu:6 -n 1 --mem=0 --cpus-per-task=44 --pty -t 2:00:00 bash'
+# allocate an interactive shell for two hours
+#   getNode 2  # allocates 2 interactive nodes (default: 1)
+function getNode() {
+    if [ -z "$1" ] ; then
+        numNodes=1
+    else
+        numNodes=$1
+    fi
+    export OMP_NUM_THREADS=7
+    srun --time=2:00:00 --nodes=$numNodes --ntasks=$((6 * $numNodes)) --ntasks-per-node=7 --cpus-per-task=7 --mem=0 --exclusive --gres=gpu:6 -p ml --pty bash
```

Comment:
here you could add `--exclusive` as well

Reply:
Yes because we do not like to share the node

Reply:
sorry